### PR TITLE
remove obsolete classes/methods from signal r

### PR DESCRIPTION
**Pull request by dpr-dev**
[#10756](https://github.com/aspnet/AspNetCore/issues/10756)

### CODE REVIEW

File: src/SignalR/server/SignalR/test/MapSignalRTests.cs [hunk 1]
```diff
@@ -14,9 +14,9 @@ public class MapSignalRTests
         [Fact]
         public void MapSignalRFailsForInvalidHub()
         {
-            var ex = Assert.Throws<NotSupportedException>(() =>
+            NotSupportedException ex = Assert.Throws<NotSupportedException>(() =>
```
Comment (halter73) [inline src/SignalR/server/SignalR/test/MapSignalRTests.cs]: Is an auto-formatter going overboard with un-varing declarations?

**Comment by BrennanConroy** [comment]
Sorry, we're not removing these until our **NEXT** major release. We literally just marked them obsolete in this release (3.0).

**Comment by halter73** [comment]
Yep. We're still using the master branch for future 3.0 previews, 3.0 rtm, 3.1, etc...

**Comment by dpr-dev** [comment]
I did not notice the information about "next major version", sorry

**Comment by Kahbazi** [comment]
@dpr-dev Since master branch if for v5.0,  now is probably a good time to reopen this PR.

**Comment by anurse** [comment]
I would suggest just opening a new one, since the `master` branch has diverged quite a bit and rebasing will be a challenge but yep, it's reasonable to consider this change in 5.0 now.

**Comment by dpr-dev** [comment]
ok, i will do it

**Pull request closed without merging**